test/test_csv_parsing.rb

```diff
@@ -189,6 +189,11 @@ def test_field_size_limit_controls_lookahead
                              field_size_limit: 2048 )
   end
 
+  def test_line_size_limit
+    assert_parse_errors_out( 'valid,fields,"' + BIG_DATA + '"',
+                             max_line_length_bytes: 8 )
```

Comment:
I'm not a huge fan of using the same error class everywhere and not checking the error message in the assertions. There's a real chance of false positives - some other constraint might raise this error and this spec would pass.

See if you could incorporate `assert_raise_message` into the `assert_parse_errors_out` method

Reply:
Added a assertion for the message